bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5289,8 +5754,12 @@ msgstr "Seleccionar el equipo"
 
 #: bika/lims/browser/worksheet/views/add_blank.py:32
 #: bika/lims/browser/worksheet/views/add_control.py:31
-msgid "Select services in the left column to locate reference samples. Select a reference by clicking it. "
-msgstr "Seleccione los servicios de la columna de la izquierda para encontrar muestras de referencia. Podréis seleccionarlas haciendo clic encima."
+msgid ""
+"Select services in the left column to locate reference samples. Select a "
+"reference by clicking it. "
+msgstr ""
+"Seleccione los servicios de la columna de la izquierda para encontrar "
+"muestras de referencia. Podréis seleccionarlas haciendo clic encima."
```

Comment:
@xispa I would say _Podrá_ instead of _Podréis_ to be consistent with how we treat the user.